tests/unit_tests/test_helm_chart.py

```diff
@@ -305,3 +293,71 @@ def group_manifests(ungrouped: Iterable[Mapping[str, Any]]) -> GroupedManifests:
             )
         group[name] = manifest
     return groups
+
+
+def test_init_container_config_not_available_when_disabled():
+    manifests = render_chart(
+        values={
+            "resources": HIGH_RESOURCES,
+            "initResources": LOW_RESOURCES,
+            "initContainer": {"enabled": False},
+        }
+    )
+
+    assert (
+        (
+            manifests["StatefulSet"]["blueapi"]["spec"]["template"]["spec"][
+                "initContainers"
+            ]
+        )
+        is None
+    )
+
+    config = yaml.safe_load(
+        manifests["ConfigMap"]["blueapi-config"]["data"]["config.yaml"]
+    )
+
+    assert config.get("scratch", None) is None
```

Comment:
Should: Be specific, either you're expecting an empty scratch block or no scratch block. The test should fail if you get the one you're not expecting, so

```suggestion
    assert "scratch" not in config
```

Or 

```suggestion
    assert config["scratch"] is None
```

I think the first one?